lib/tasks/admin.rake

```diff
@@ -0,0 +1,15 @@
+namespace :admin do
```

Comment:
This is a command line task that an admin can run from the command line whenever they want.

It's documented [here][1], and it aims to replace the instructions that were initially written [here][2] that required people to use `irb` to enter the code themselves.

[1]: https://github.com/autolab/Autolab/wiki/Deploying-Autolab-with-Docker#13-create-an-administrator-account
[2]: https://github.com/autolab/Autolab/wiki/Deploying-Autolab-on-Ubuntu#optional-setup-administrator